ocaml/xapi/xapi_pool_transition.ml

```diff
@@ -206,7 +206,8 @@ let become_another_masters_slave master_address =
     debug "Setting pool.conf to point to %s" master_address ;
     set_role new_role ;
     run_external_scripts false ;
-    Xapi_fuse.light_fuse_and_run ()
+    Xapi_fuse.light_fuse_and_run () ;
```

Comment:
This gives the next functions 5 seconds to complete until it is cancelled, are you sure you don't want to clean up the the repository _before_ this call?

Reply:
Aah, I see. This `fuse` will restart the toolstack. Thanks.